code/game/objects/structures/ai_core.dm

```diff
@@ -144,7 +144,7 @@
 					user << "<span class='warning'>This MMI does not seem to fit!</span>"
 					return
 
-				if(M.hacked)
+				if(M.hacked || M.clockwork)

```

Comment:
this is butts and I hate it but it's A technically a balance thing and B i don't feel like refactoring all this code to work with mmi laws right now